src/store/actionCreators.tsx

```diff
@@ -0,0 +1,36 @@
+import { ActionType } from "./actionTypes";
+
+export const resetMovieActionCreator = (movie: string) => {
```

Comment:
please specify type of return value for all action creators too, use types from action.js